### DISCUSSION TITLE
Cefsharp upgrade to 115.4.0 version doensot render the web content, earlier version 83.0.1 worked fine with a WPF application

### DISCUSSION THREAD

**Discussion by AanchalTewary** [cefsharp/CefSharp#5006] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

115.4.0

### Operating System

Windows 10

### Architecture

x86

### .Net Version

.Net 4.8

### Implementation

WPF

### Reproduction Steps

We upgraded the Cefsharp version and used paltform target x84 to run it and it doesnot render the web page anymore. Earlier version 83.0.1 worked fine.

### Expected behavior

It should launch the web page.

### Actual behavior

It shows a black screen and the window doesnot even get listed in the cef debugger localhost:9999

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

### Other information

Please advise what might be wrong

**Comment by amaitland** [comment]
> What version are you using? Please only open an issue if you can reproduce the problem with version 130.1.90 or later.

As per https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

The https://github.com/cefsharp/CefSharp.MinimalExample is working fine with the current supported version (M131 at time of writing).

M115 is old and unsupported.

**Reply by amaitland** [reply]
If you are having issues with the current supported version then please use https://github.com/cefsharp/CefSharp/discussions to ask for help.
